projects/storefrontlib/cms-components/misc/icon/icon-loader.service.spec.ts

```diff
@@ -7,7 +8,8 @@ import { IconConfig, IconResourceType, ICON_TYPE } from './icon.model';
 
 const FONT_AWESOME_RESOURCE =
   'https://use.fontawesome.com/releases/v5.8.1/css/all.css';
-const MockFontIconConfig: IconConfig = {
+
+export const MockIconConfig: IconConfig = {
```

Comment:
Exported this, so it can be reused in the IconComponent spec.

Not sure, if this is good practice. But it seems preferable to use a common `IconConfig` rather than setting up custom mocks or spies for each individual test.

Maybe move it to somewhere else? E.g. to the icon-testing module?